bitnami/spark/templates/statefulset-worker.yaml

```
@@ -249,6 +254,8 @@ spec:
               value: {{ .Values.security.ssl.needClientAuth | quote }}
             - name: SPARK_SSL_PROTOCOL
               value: {{ .Values.security.ssl.protocol | quote }}
+            - name: SPARK_WEBUI_SSL_PORT
+              value: {{ .Values.worker.webPortHttps }}
```

Missing quotes will cause the Statefulset to fail
```suggestion
              value: {{ .Values.worker.webPortHttps | quote }}
```